rbc/rbclib/intrinsic.py

```diff
@@ -0,0 +1,23 @@
+from llvmlite import ir
+from numba.core import extending
+from numba.core import types as nb_types
+from rbc import irutils
+
+
+@extending.intrinsic
+def add_ints(typingctx, a_type, b_type):
+    if (a_type, b_type) != (nb_types.int64, nb_types.int64):
+        return None
```

Comment:
I did `return None` because the [numba docs](https://numba.pydata.org/numba-doc/latest/extending/high-level.html#implementing-intrinsics) contains the following:

> The return values of func should be a 2-tuple of expected type signature, and a code-generation function that will passed to lower_builtin(). For an unsupported operation, return None.

What is the different between `return None` and `raise NumbaTypeError()`? When should I use one instead of the other?

Reply:
They both work. But in a type error, you explicitly say why the function didn't match. I guess for the tests, this is not needed since you call the function with the correct argument types.

Reply:
thanks, fixed by 3b621d0